docs/using-airbyte/core-concepts/typing-deduping.md

```diff
@@ -76,13 +103,37 @@ You also now see the following changes in Airbyte-provided columns:
 
 ![Airbyte Destinations V2 Column Changes](../../release_notes/assets/updated_table_columns.png)
 
+## On final table re-creation
+
+From time to time, Airbyte will drop and re-create the final table produced by a sync. This is done
+as transitionally as possible, and should be invisible to most observers. This is done for a number
+of reasons, including:
+
+- **Schema Migrations** - Many destinations lack the ability to control column order, or cannot
+  alter one data type to another. Re-creating the table allows Airbyte to strictly control the
+  column order and data types of each column.
+- **Transactional Full Refresh** - In order to keep your final tables consistently useful, when a
+  refresh or reset occurs, airbyte will erase the raw tables, and then build a new tmp final table
+  first. Only at hte last second will Airbyte swap the old and tmp final tables, usually via a
+  rename. Otherwise, there would be a period of time where the final table is empty, which could
```

Comment:
```suggestion
  first and performs an atomic swap of the old and tmp final tables, usually via a
  rename. Otherwise, there would be a period of time where the final table is empty, which could
```